actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -58,56 +58,38 @@ module TranslationHelper
       # when you call translate in a template and translators know which keys
       # they can provide HTML values for.
       def translate(key, **options)
-        unless options[:default].nil?
-          remaining_defaults = Array.wrap(options.delete(:default)).compact
-          options[:default] = remaining_defaults unless remaining_defaults.first.kind_of?(Symbol)
-        end
+        return key.map { |k| translate(k, **options) } if key.is_a?(Array)
```

Comment:
Do we have to update the docs for this change? Also I'm wondering what the motivation for this change is. Is it to make code like the following simpler? 

```
translate([:apple, :pear]).join(", ")
```

instead of

```
"#{translate(:apple)}, #{translate(:pear)}"
```

Reply:
This is to support [bulk lookup](https://github.com/ruby-i18n/i18n/blob/v1.6.0/lib/i18n.rb#L149-L158).  I doubt it's commonly used, but basic usage is currently supported:

```ruby
helper.translate([:hello, :hello])
# => ["Hello world", "Hello world"]

helper.translate([:missing, :missing], default: "Hello world")
# => ["Hello world", "Hello world"]
```

However a few corner cases are broken:

```ruby
helper.translate([:missing, :missing], default: :hello)
# => "Hello world"

helper.translate([:hello_html, :hello_html]).all?(&:html_safe?)
# => false
```

Since `translate` is documented as ["Delegates to `I18n.translate`"](https://github.com/rails/rails/blob/8654891cc7d0ea80b67ef7d352cc81463f526532/actionview/lib/action_view/helpers/translation_helper.rb#L19-L20), I don't think we need to add further documentation.

Alternatively, if we don't want to support this feature, I can remove this line.  I could also raise an `ArgumentError` when an array is passed.